src/tree/v2/application/SubscriptionItem.ts

```diff
@@ -20,7 +29,13 @@ export class SubscriptionItem implements ResourceGroupsItem {
     }
 
     async getChildren(): Promise<ResourceGroupsItem[]> {
-        const resources = await this.resourceProviderManager.getResources(this.subscription);
+        let resources = await this.resourceProviderManager.getResources(this.subscription);
+
+        const showHiddenTypes = settingUtils.getWorkspaceSetting<boolean>(showHiddenTypesSettingKey);
+
+        if (!showHiddenTypes) {
+            resources = resources.filter(resource => resource.azureResourceType.type === 'microsoft.resources/resourcegroups' || (resource.resourceType && supportedResourceTypes.find(type => type === resource.resourceType)));
```

Comment:
(Same as above) Do we not have an `AzExtResourceType` for RGs?

Reply:
Not yet, but added an [issue](https://github.com/microsoft/vscode-azuretools/issues/1244) for them to be added to the enumeration.